diff --git a/app/scripts/models/app-settings-model.js b/app/scripts/models/app-settings-model.js
index eebf08ff..a23e475a 100644
--- a/app/scripts/models/app-settings-model.js
+++ b/app/scripts/models/app-settings-model.js
@@ -29,7 +29,7 @@ class AppSettingsModel extends Model {
 
 AppSettingsModel.defineModelProperties(
     {
-        theme: 'fb',
+        theme: 'wh',
         locale: null,
         expandGroups: true,
         listViewWidth: null,
@@ -56,28 +56,30 @@ AppSettingsModel.defineModelProperties(
         skipOpenLocalWarn: false,
         hideEmptyFields: false,
         skipHttpsWarning: false,
-        demoOpened: false,
+        demoOpened: true,
         fontSize: 0,
         tableViewColumns: null,
         generatorPresets: null,
         generatorHidePassword: false,
         cacheConfigSettings: false,
-        allowIframes: false,
+        allowIframes: true,
 
         canOpen: true,
-        canOpenDemo: true,
+        canOpenDemo: false,
         canOpenSettings: true,
-        canCreate: true,
+        canCreate: false,
         canImportXml: true,
         canImportCsv: true,
         canRemoveLatest: true,
         canExportXml: true,
         canExportHtml: true,
 
-        dropbox: true,
+        dropbox: false,
         webdav: true,
-        gdrive: true,
-        onedrive: true
+        gdrive: false,
+        onedrive: false,
+
+        webdavSaveMethod: 'put'
     },
     { extensions: true }
 );
