diff --git a/Gruntfile.js b/Gruntfile.js
index 595a470..863f378 100644
--- a/Gruntfile.js
+++ b/Gruntfile.js
@@ -17,7 +17,7 @@ module.exports = function(grunt) {
     var dt = new Date().toISOString().replace(/T.*/, '');
     var minElectronVersionForUpdate = '1.0.1';
     var zipCommentPlaceholder = 'zip_comment_placeholder_that_will_be_replaced_with_hash';
-    var electronVersion = pkg.devDependencies['electron-prebuilt'].replace(/^\D/, '');
+    // var electronVersion = pkg.devDependencies['electron-prebuilt'].replace(/^\D/, '');
 
     while (zipCommentPlaceholder.length < 512) {
         zipCommentPlaceholder += '.';
@@ -303,50 +303,50 @@ module.exports = function(grunt) {
                 tasks: ['copy:html']
             }
         },
-        electron: {
-            options: {
-                name: 'KeeWeb',
-                dir: 'tmp/desktop/app',
-                out: 'tmp/desktop',
-                version: electronVersion,
-                overwrite: true,
-                'app-copyright': 'Copyright © 2016 Antelle',
-                'app-version': pkg.version,
-                'build-version': '<%= gitinfo.local.branch.current.shortSHA %>'
-            },
-            linux: {
-                options: {
-                    platform: 'linux',
-                    arch: ['x64', 'ia32'],
-                    icon: 'graphics/icon.ico'
-                }
-            },
-            darwin: {
-                options: {
-                    platform: 'darwin',
-                    arch: ['x64'],
-                    icon: 'graphics/icon.icns',
-                    'app-bundle-id': 'net.antelle.keeweb',
-                    'app-category-type': 'public.app-category.productivity',
-                    'extend-info': 'package/osx/extend.plist'
-                }
-            },
-            win32: {
-                options: {
-                    platform: 'win32',
-                    arch: ['ia32', 'x64'],
-                    icon: 'graphics/icon.ico',
-                    'build-version': pkg.version,
-                    'version-string': {
-                        'CompanyName': 'KeeWeb',
-                        'FileDescription': pkg.description,
-                        'OriginalFilename': 'KeeWeb.exe',
-                        'ProductName': 'KeeWeb',
-                        'InternalName': 'KeeWeb'
-                    }
-                }
-            }
-        },
+        // electron: {
+        //     options: {
+        //         name: 'KeeWeb',
+        //         dir: 'tmp/desktop/app',
+        //         out: 'tmp/desktop',
+        //         version: electronVersion,
+        //         overwrite: true,
+        //         'app-copyright': 'Copyright © 2016 Antelle',
+        //         'app-version': pkg.version,
+        //         'build-version': '<%= gitinfo.local.branch.current.shortSHA %>'
+        //     },
+        //     linux: {
+        //         options: {
+        //             platform: 'linux',
+        //             arch: ['x64', 'ia32'],
+        //             icon: 'graphics/icon.ico'
+        //         }
+        //     },
+        //     darwin: {
+        //         options: {
+        //             platform: 'darwin',
+        //             arch: ['x64'],
+        //             icon: 'graphics/icon.icns',
+        //             'app-bundle-id': 'net.antelle.keeweb',
+        //             'app-category-type': 'public.app-category.productivity',
+        //             'extend-info': 'package/osx/extend.plist'
+        //         }
+        //     },
+        //     win32: {
+        //         options: {
+        //             platform: 'win32',
+        //             arch: ['ia32', 'x64'],
+        //             icon: 'graphics/icon.ico',
+        //             'build-version': pkg.version,
+        //             'version-string': {
+        //                 'CompanyName': 'KeeWeb',
+        //                 'FileDescription': pkg.description,
+        //                 'OriginalFilename': 'KeeWeb.exe',
+        //                 'ProductName': 'KeeWeb',
+        //                 'InternalName': 'KeeWeb'
+        //             }
+        //         }
+        //     }
+        // },
         compress: {
             options: {
                 level: 6
