diff --git a/Gruntfile.js b/Gruntfile.js
index ec5a2f7d..682f4a0a 100644
--- a/Gruntfile.js
+++ b/Gruntfile.js
@@ -30,9 +30,7 @@ module.exports = function (grunt) {
     const zipCommentPlaceholder =
         zipCommentPlaceholderPart + '.'.repeat(512 - zipCommentPlaceholderPart.length);
     const electronVersion = pkg.dependencies.electron.replace(/^\D/, '');
-    const skipSign = grunt.option('skip-sign');
-    const getCodeSignConfig = () =>
-        skipSign ? { identities: {} } : require('./keys/codesign.json');
+    const getCodeSignConfig = { identities: {} };
 
     const sha = execSync('git rev-parse --short HEAD').toString('utf8').trim();
 
