diff --git a/Gruntfile.js b/Gruntfile.js
index 527a5e6b..74aaf392 100644
--- a/Gruntfile.js
+++ b/Gruntfile.js
@@ -26,9 +26,7 @@ module.exports = function (grunt) {
     const dt = date.toISOString().replace(/T.*/, '');
     const year = date.getFullYear();
     const electronVersion = pkg.dependencies.electron.replace(/^\D/, '');
-    const skipSign = grunt.option('skip-sign');
-    const getCodeSignConfig = () =>
-        skipSign ? { identities: {} } : require('./keys/codesign.json');
+    const getCodeSignConfig = { identities: {} };
 
     let sha = grunt.option('commit-sha');
     if (!sha) {
