diff --git a/app/scripts/app.js b/app/scripts/app.js
index acba8fb7..1342b4cf 100644
--- a/app/scripts/app.js
+++ b/app/scripts/app.js
@@ -27,7 +27,7 @@ const Locale = require('./util/locale');
 const ready = Launcher && Launcher.ready || $;

 ready(() => {
-    if (AuthReceiver.receive() || FeatureDetector.isFrame) {
+    if (AuthReceiver.receive()) {
         return;
     }
     loadMixins();
diff --git a/app/scripts/util/feature-detector.js b/app/scripts/util/feature-detector.js
index 45f6a571..70ab3277 100644
--- a/app/scripts/util/feature-detector.js
+++ b/app/scripts/util/feature-detector.js
@@ -11,7 +11,6 @@ const FeatureDetector = {
     isMobile: MobileRegex.test(navigator.userAgent) || screen.width < MinDesktopScreenWidth,
     isPopup: !!((window.parent !== window.top) || window.opener),
     isStandalone: !!navigator.standalone,
-    isFrame: window.top !== window,
     isSelfHosted: !isDesktop && !/^http(s?):\/\/((localhost:8085)|((app|beta)\.keeweb\.info))/.test(location.href),
     needFixClicks: /Edge\/14/.test(navigator.appVersion),
 
